src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -0,0 +1,360 @@
+
+                <Button
+                  loading={isSubmitting}
+                  width="100%"
+                  onClick={() => handleSubmit()}
+                  disabled={isSaveAlertButtonDisabled}
+                >
+                  Save Alert
+                </Button>
+
+                <Spacer mt={1} />
+
+                <Button
+                  variant="secondaryOutline"
+                  width="100%"
+                  onClick={onDeleteClick}
+                >
+                  Delete Alert
+                </Button>
+              </Media>
+            </Join>
+          </Box>
+        )
+      }}
+    </Formik>
+  )
+}
+
+export const SavedSearchAlertEditFormFragmentContainer = createFragmentContainer(
+  SavedSearchAlertEditForm,
+  {
+    savedSearch: graphql`
+      fragment SavedSearchAlertEditForm_savedSearch on SearchCriteria {
+        internalID
+        acquireable
+        additionalGeneIDs
+        artistIDs
+        atAuction
+        attributionClass
+        colors
+        dimensionRange
+        sizes
+        height
+        inquireableOnly
+        internalID
+        locationCities
+        majorPeriods
+        materialsTerms
+        offerable
+        partnerIDs
+        priceRange
+        userAlertSettings {
+          name
+          email
+          push
+        }
+        width
+      }
+    `,
+    artist: graphql`
+      fragment SavedSearchAlertEditForm_artist on Artist {
+        internalID
+        name
+        slug
+      }
+    `,
+    artworksConnection: graphql`
+      fragment SavedSearchAlertEditForm_artworksConnection on FilterArtworksConnection {
+        aggregations {
+          slice
+          counts {
+            count
+            name
+            value
+          }
+        }
+      }
+    `,
+  }
+)
+
+const SAVED_SEARCH_ALERT_EDIT_FORM_QUERY = graphql`
+  query SavedSearchAlertEditFormQuery($id: ID!, $artistId: String!) {
+    me {
+      savedSearch(id: $id) {
+        ...SavedSearchAlertEditForm_savedSearch
+      }
+    }
+    artist(id: $artistId) {
+      ...SavedSearchAlertEditForm_artist
+    }
+    artworksConnection(
+      first: 0
+      artistID: $artistId
+      aggregations: [
+        ARTIST
+        LOCATION_CITY
+        MATERIALS_TERMS
+        MEDIUM
+        PARTNER
+        COLOR
+      ]
+    ) {
+      ...SavedSearchAlertEditForm_artworksConnection
+    }
+  }
+`
+
+export const SavedSearchAlertEditFormQueryRenderer: React.FC<SavedSearchAlertEditFormQueryRendererProps> = ({
+  editAlertEntity,
+  onDeleteClick,
+  onCompleted,
+}) => {
+  return (
+    <SystemQueryRenderer<SavedSearchAlertEditFormQuery>
+      query={SAVED_SEARCH_ALERT_EDIT_FORM_QUERY}
+      variables={{ id: editAlertEntity.id, artistId: editAlertEntity.artistId }}
+      placeholder={<SavedSearchAlertEditFormPlaceholder />}
+      cacheConfig={{ force: true }}
+      render={({ props, error }) => {
+        if (error) {
+          console.error(error)
```

Comment:
question(non-blocking): shouldn't we keep track of the failed queries in sentry here or console.error is enough?

Reply:
I did by analogy how errors are handled when using `SystemQueryRenderer` in other places ([example #1](https://github.com/artsy/force/blob/8d6e5277b254d4aa70cb7428c5986b4e56b21e82/src/v2/Apps/Artwork/Components/ArtistInfo.tsx#L220-L223), [example #2](https://github.com/artsy/force/blob/8d6e5277b254d4aa70cb7428c5986b4e56b21e82/src/v2/Apps/Article/Components/ArticleVerticalRelatedArticles.tsx#L117-L120), [example #3](https://github.com/artsy/force/blob/8d6e5277b254d4aa70cb7428c5986b4e56b21e82/src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGallery.tsx#L217-L220)). @damassi should we use `logger.error` here instead of `console.error`?